README.md

```diff
@@ -1,3 +1,99 @@
+<!-- PROJECT LOGO -->
+
+<p align="center">
+  <a href="https://openliberty.io/">
+    <img src="https://openliberty.io/img/spaceship.svg" alt="Logo">
+  </a>
+</p>
+<p align="center">
+  <a href="https://openliberty.io/">
+    <img src="https://github.com/OpenLiberty/open-liberty/blob/master/logos/logo_horizontal_light_navy.png" alt="title" width="400">
+  </a>
+</p>
+<br />
+
+
+[![License](https://img.shields.io/badge/License-ASL%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
+
+# Summary
+
+A devfile-based application stack for Open Liberty built with Gradle
+
 # Open Liberty Application Stack
 
-WIP
+The Open Liberty application stack provides a consistent way of developing microservices based upon the [Jakarta EE](https://jakarta.ee/) and [Eclipse MicroProfile](https://microprofile.io) specifications. This stack lets you use [Gradle](https://gradle.org/) to develop applications for [Open Liberty](https://openliberty.io) runtime, that is running on OpenJDK with container-optimizations in OpenJ9.
+
+This stack is based on OpenJDK with container-optimizations in OpenJ9 and Open Liberty. It provides live reloading during development by utilizing the ["dev mode"](https://openliberty.io/blog/2019/10/22/liberty-dev-mode.html) capability in the liberty-gradle-plugin.  
+
+**Note:** Gradle is provided by the stack, allowing you to build, test, and debug your Java application without installing Gradle locally.
+
+## Getting Started With the Intro Sample
+
+> It's recommended to use the latest version of OpenShift Do (odo). You can install odo using [these instructions](https://odo.dev/docs/installing-odo/)
+
+1. Perform an `oc login` to your cluster.
+
+1. Clone the application-stack-intro repository
+
+    ```shell
+    git clone git@github.com:OpenLiberty/application-stack-intro.git
```

Comment:
So we're adding build.gradle to intro?    We didn't do that yet though.

Reply:
You are correct, this needed to cleaned up along with other references to doc that are currently not updated.  
I will update this and a few of the other things from your review.